docs/installation/linux/ubuntulinux.md

```diff
@@ -251,7 +251,7 @@ To create the `docker` group and add your user:
 
 3. Add your user to `docker` group.
 
-        $ sudo usermod -aG docker ubuntu
+        $ sudo usermod -aG docker $USER
```

Comment:
line 246 no longer applies then (https://github.com/docker/docker/pull/25239/files#diff-26ed3ddfd9bfa4b4334997b3d9bcad44R246)

perhaps change `ubuntu` to `some-user-name` (instead of using the `$USER` variable),
or remove the line at 246